dist/init/systemd/rkt-api-tcp.socket

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=rkt api service socket
+PartOf=rkt-api.service
+
+[Socket]
+ListenStream=15441
```

Comment:
The default should not to listen on all interfaces but only on "lo":
```
ListenStream=127.0.0.1:15441
ListenStream=[::1]:15441
```

But does socket activation actually work? It does not seem to have the code for it:
https://github.com/coreos/rkt/blob/master/rkt/api_service.go#L673